browser/scripts/ui/playerUI.js

```diff
@@ -308,8 +316,8 @@ var VizorPlayerUI = function() {
 		}
 
 		// dibs on these
-		document.body.addEventListener('touchend', headerHandler, true)
-		document.body.addEventListener('mouseup', headerHandler, true)
+		document.body.addEventListener('touchend', headerHandler, false)
+		document.body.addEventListener('mouseup', headerHandler, false)
```

Comment:
can remove dibs comment now? also false is the default for .addEventListener so it can go too.